argparse.go

```diff
@@ -266,7 +310,7 @@ func (o *Command) Happened() bool {
 //
 // Accepts an interface that can be error, string or fmt.Stringer that will be prepended to a message.
 // All other interface types will be ignored
-func (o *Command) Usage(msg interface{}) string {
+func (o Command) Usage(msg interface{}) string {
```

Comment:
Should be a pointer.

Reply:
When I see a pointer caller, I assume the function will modify the caller, which is not the case for the Usage function. This could give the wrong impression to developers who use the package.

Reply:
You are right that if there's no changes then no need to pass pointer. However logic may change later and require access to internals from within the function in which case this would need to be changed back to pointer. I think it should be alright to keep as-is with passing a pointer here.